lmfdb/cluster_pictures/web_cluster_picture.py

```diff
@@ -0,0 +1,84 @@
+# -*- coding: utf-8 -*-
+from sage.all import cached_function
+
+from lmfdb import db
+from lmfdb.logger import make_logger
+wcplog = make_logger("WCP")
+
+@cached_function
```

Comment:
Why does this need to be cached?  You're not doing anything expensive here....

Reply:
Don't know. This was just copied from the number fields. I thought it was required.